dirs/dirs.go

```diff
@@ -187,6 +191,10 @@ func SetRootDir(rootdir string) {
 
 	SnapStateFile = filepath.Join(rootdir, snappyDir, "state.json")
 
+	SnapCacheDir = filepath.Join(rootdir, "/var/cache/snapd")
```

Comment:
We need to fix packaging so there's proper ownership on this directory on all distributions, and the packaging purging logic for all distros also need to account for this new directory.

Reply:
:heart: - indeed

Reply:
gah. you are correct. See, this is (one of the reasons) why I'd originally had it under `/var/lib/snapd` -- laziness.

Reply:
Yeah, to be honest I wouldn't mind having these files inside /var/lib/snapd either.